bobtimus/src/loan.rs

```diff
@@ -403,27 +408,34 @@ fn validate_loan_is_acceptable(
         }));
     }
 
-    // If no collateraliztion thresholds are specified in the offer then we ignore this check and only check for LTV
-    // Note that as a safety net the LTV still outweights the collateralization check.
-    if !collateralizations.is_empty() {
-        let mut sorted_collateralizations = collateralizations;
-        sorted_collateralizations.sort_by(|a, b| a.collateralization.cmp(&b.collateralization));
-        let min_collateralization = sorted_collateralizations
-            .first()
-            .context("Unable to determine minimum collateralization")?
-            .collateralization;
-
-        if request_collateralization < min_collateralization {
-            return Ok(Err(LoanValidationError::CollateralizationBelowMin {
-                request_collateralization,
-                min_collateralization,
-            }));
-        }
+    // we allow for slight price variations of 1% up or down
+    let one_percent = requested_ltv
```

Comment:
With this we have two price fluctuation intervals now - one for the actual accepted eschange rate and one for the  LTV.